db/migrate/20230329185029_add_posts_ref_to_comments.rb

```diff
@@ -0,0 +1,5 @@
+class AddPostsRefToComments < ActiveRecord::Migration[7.0]
+  def change
+    add_reference :comments, :posts, null: false, foreign_key: true
```

Comment:
- Hi :clap:, in here also, since you are referencing the `post`, it should be singular `post` and not plural `posts` because when I run your migration with `rails db:migrate`, the `schema.rb` file shows the foreign keys in plural instead of being singular. See this below:

`add_foreign_key "comments", "posts", column: "posts_id"` it should be `post_id`, kindly check the erd diagram to confirm so you can fix this.
- This should be 
```
add_reference :comments, :post, null: false, foreign_key: true
```
Let's fix this please :pray: